docs/doc_v1.0/Build_documentation.md

```diff
@@ -159,6 +160,19 @@ The _Age Data_ file assigns an age to an individual via their ID. The _Age Data_
 
 **Figure 6.** Example _Age Data_ file.
 
+#### Population Overrides
+
+The population overrides file provides the user with the ability to modify the population properties. These fall into two categories:
+
+1. The age dependent health modifier - This is a list containing an age range and a maximum health for someone in that age. 
+This value is multiplied by a random number between 0 and 1 to give the final age of each person.
```

Comment:
Did you mean the following?

This value is multiplied by a random number between 0 and 1 and is assigned to each individual to represent their health.

```suggestion
This value is multiplied by a random number between 0 and 1 to give the final age of each person.
```